src/admin/vote/Assign.jsx

```
@@ -446,19 +503,31 @@
         </div>
       </div>
       <p />
-      Hier sind die optimierten Zuordnungen der Schüler zu den Projekten.
+      {mode !== "power-search" && (
+        <>
+          Insgesamt wurden {Object.keys(results).length} Schüler zugeordnet.
+          Davon haben:{" "}
+          {Object.entries(wahlenCounts).map(([wahl, count]) => (
+            <>
+              <i>{count}</i> Schüler die{" "}
+              {wahl === "1"
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)